tests/runtime-size.spec.ts

```diff
@@ -10,14 +10,14 @@ describe('runtime size', () => {
     await clearDir(outputDirPath);
   });
 
-  it('should throw when lib size exceeds 27kb', () => {
+  it('should throw when lib size exceeds 30kb', () => {
     return runWebpack({
       output: {
         path: path.resolve(outputDirPath),
         libraryTarget: 'commonjs',
       },
       performance: {
-        maxEntrypointSize: 27 * 1024,
+        maxEntrypointSize: 30 * 1024,
```

Comment:
so it means that every bundle will increase in 3kb? :(

Reply:
It should be compensated by reducing `fallback` in the user's codebase :)

Reply:
yeah but only once all projects will migrate to editor flow + tpa settings

Reply:
but we don't bundle it in the lean version afair so ok